test/embed.test.ts

```diff
@@ -93,3 +93,22 @@ test('guessMode from Vega-Lite spec', () => {
 test('guessMode from Vega spec', () => {
   expect(guessMode({ marks: [] }, 'invalid' as Mode)).toBe('vega');
 });
+
+test('can change i18n strings', async () => {
+  const i18n = {COMPILED_ACTION: 'foo', EDITOR_ACTION: 'bar', PNG_ACTION: 'baz', SOURCE_ACTION: 'qux', SVG_ACTION: 'quux'};
+  const { COMPILED_ACTION, EDITOR_ACTION, PNG_ACTION, SOURCE_ACTION, SVG_ACTION } = i18n;
+  const el = document.createElement('div');
+  let ctrl;
+  let ctrlChildren;
+  await embed(el, vlSpec, {actions: true, i18n});
+
+  ctrl = el.children[2].children[0];
+  ctrlChildren = ctrl.children;
+
+  expect(ctrl.childElementCount).toBe(5);
```

Comment:
In the test, please use the string and not a variable .

Reply:
Changes made. Please check.